src/applications/disability-benefits/all-claims/content/fullyDevelopedClaim.jsx

```diff
@@ -8,7 +8,10 @@ export const FDCDescription = (
       uploaded all the supporting documents or supplemental forms needed to
       support your claim.
     </p>
-    <a href="/pension/how-to-apply/fully-developed-claim/" target="_blank">
+    <a
+      href="/disability/how-to-file-claim/evidence-needed/fully-developed-claims/"
+      target="_blank"
```

Comment:
Should this also have the `rel` per your other PR?

Reply:
🤔 I don't think so, this seems to not be a teamsite link